amundsen_application/static/js/features/ColumnList/ColumnLineage/index.tsx

```diff
@@ -0,0 +1,90 @@
+// Copyright Contributors to the Amundsen project.
+// SPDX-License-Identifier: Apache-2.0
+
+import { connect } from 'react-redux';
+
+import { GlobalState } from 'ducks/rootReducer';
+import * as React from 'react';
+import { emptyLineage } from 'ducks/tableMetadata/reducer';
+
+import { getColumnLineageLink } from 'config/config-utils';
+import './styles.scss';
+import { Lineage, TableMetadata } from 'interfaces/TableMetadata';
+
+interface ComponentProps {
+  columnName: string;
+  tableKey: string;
+}
+
+interface StateFromProps {
+  columnLineage: Lineage;
+  tableData: TableMetadata;
+}
+
+type Props = ComponentProps & StateFromProps;
+
+const getLink = (table) =>
+  `/table_detail/${table.cluster}/${table.database}/${table.schema}/${table.name}` +
+  `?source=column_lineage`;
+
+export class ColumnLineageList extends React.Component<Props> {
+  renderLineageLinks(entity, index) {
+    if (index >= 5) {
```

Comment:
Why 5? Could we make this number a constant with a name that explains itself?

Reply:
You're right, I forgot to put this into a constant.